### PR TITLE
Update update-to-head.sh to latest version

### PR DESCRIPTION
**Pull request by alanfx**

### CODE REVIEW

File: openshift/release/update-to-head.sh [hunk 1]
```diff
@@ -1,19 +1,30 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (vdemeester) [inline openshift/release/update-to-head.sh]: :heart_eyes_cat:

File: openshift/release/update-to-head.sh [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
+# Reset release-next to upstream/master.
 git fetch upstream master
-git rebase upstream/master
+git checkout upstream/master -B release-next
 
 # Update openshift's master and take all needed files from there.
 git fetch openshift master
 git checkout openshift/master openshift OWNERS_ALIASES OWNERS Makefile
 make generate-dockerfiles
+make RELEASE=ci generate-release
```
Comment (vdemeester) [inline openshift/release/update-to-head.sh]: ah, maybe the `Makefile` needs updates too, as I don't see this target in ours

File: openshift/release/resolve.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  for yaml in "$dir"/*.yaml; do
+    echo "---" >> $resolved_file_name
+    # 1. Prefix test image references with test-
+    # 2. Rewrite image references
+    # 3. Update config map entry
+    # 4. Remove comment lines
+    # 5. Remove empty lines
+    sed -e "s+\(.* image: \)\(github.com\)\(.*/\)\(test/\)\(.*\)+\1\2 \3\4test-\5+g" \
+        -e "s+\(.* image: \)\(github.com\)\(.*/\)\(.*\)+\1 ${image_prefix}\4${image_tag}+g" \
```
Comment (chmouel) [inline openshift/release/resolve.sh]: 💅

**Pull request merged**